evm_arithmetization/tests/two_to_one_block.rs

```diff
@@ -168,12 +169,15 @@ fn test_two_to_one_block_aggregation() -> anyhow::Result<()> {
     let some_timestamps = [127, 42, 65, 43];
 
     let all_stark = AllStark::<F, D>::default();
-    let config = StarkConfig::standard_fast_config();
+    let config = TEST_STARK_CONFIG;
 
     let all_circuits = AllRecursiveCircuits::new(
         &all_stark,
         &[16..17, 8..9, 12..13, 8..9, 8..9, 6..7, 17..18, 16..17, 7..8],
         &config,
+        Some(&TEST_RECURSION_CONFIG),
+        Some(&TEST_RECURSION_CONFIG),
+        None,
```

Comment:
Why isn't it `TWO_TO_ONE_BLOCK_CIRCUIT_TEST_THRESHOLD_DEGREE_BITS`?

Reply:
This is only for `2-to-1 block` circuit's threshold.

Reply:
Yes, we should use `TEST_THRESHOLD_DEGREE_BITS`. Thanks for the catch!

Reply:
Now, we have two_to_one_block aggregation test passing in 38 seconds on my MacBook. We can confidently remove [ignored] from it now.